.golangci.yml

```diff
@@ -108,3 +108,6 @@ issues:
 
   exclude-dirs:
     - licenses
+
+  exclude:
+    - "SA1019: (.+)Secret is deprecated: Use AdditionalSecrets instead."
```

Comment:
Can you add a comment why this is necessary?

Reply:
Yup, will add a comment line above this.

It's because the linter fails, since we are still using the deprecated field in a couple of places.

Reply:
Added the comment and updated exclude rule to include only the 2 files where the deprecated field is actually used.

This way if somebody accidentally uses the deprecated Secret field in some other part of code, they will get the linter error.